tests/ci-helper.test.ts

```diff
@@ -304,6 +311,7 @@ testQ("handle comment allow fail invalid user", async () => {
     };
 
     ci.setGHGetPRComment(comment);
+    ci.letGHGetGitHubUserInfoThrow("is not a valid GitHub username");
```

Comment:
Would any error string work here?  ie, does a specific string need to be specified for the throw?  The error message is set by CIHelper.  This suggests the error is returned (in the throw) by Octokit.

Sorry I can't test this at the moment.  Windows 24H2 may have given me bad drivers and am slowing rebuilding in between system freezes.

Reply:
Yes, any error string would work here! Will replace with "SOME ERROR" ;-)

Reply:
> Windows 24H2 may have given me bad drivers and am slowly rebuilding in between system freezes.

fyi, the freezing appears to have been caused by the power management driver (shutting down the wrong cpu?).  There is a registry setting to turn it off.